img2mol/inference.py

```diff
@@ -12,20 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
Could you move this to the actual location where the warning is emitted and guard via `warnings.catch_warnings` context manager? I guess this happens due to old versions running via py3.6?

Reply:
Thanks for the tip, I did not know warnings.catch_warnings before! :)
The warning occurs when importing when importing cddd.inference:
```
 __import__('pkg_resources').declare_namespace(__name__)
/home/otto/anaconda3/envs/img2mol/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:523: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
```

The warning is now handled via `warnings.catch_warnings`
```
try:
    with warnings.catch_warnings():
        warnings.simplefilter("ignore", FutureWarning)
        from cddd.inference import InferenceModel as CDDDInferenceModel
except ImportError:
    print("Local CDDD installation has not been found.")
```